style.css

```diff
@@ -128,6 +221,27 @@ header {
   flex-grow: 0;
 }
 
+/* @media screen and (min-width: 480px) {
+  .about-me-heading,
+  .about-me .description,
+  .connect,
+  ul.socials {
+    padding-left: 0;
+    margin-left: 0;
+  }
+} */
```

Comment:
- [OPTIONAL ]Please remove unused code in your project so that your code will look professional :wink: